### ISSUE TITLE
Request:  Separate categories and blackhole folders for Audio

### ISSUE THREAD

**Issue by DarkSir23** [DobyTang/LazyLibrarian#829]
With current Philborman release.  It would be nice to be able to have separate AudioBook categories for nzb and torrent clients, and a possible separate blackhole folder.  Additionally, a separate watch location for incoming files. 

Just a thought.  May be more of a pain.   But it would help with manually fixing failed processing.

**Comment by philborman** [comment]
I thought about that too, but then do you add a separate category for magazines, and a separate blackhole for magazines and a separate watch directory, so 3 of everything ?  It all gets a bit messy.  
Then if we add comics (no plans yet, but...)  you end up with 4 of everything.
Probably best to have all the downloaders treat them as "lazylibrarian" category and let lazylibrarian sort it out from the one folder.

If a download fails it gets renamed to whatever_name.fail  so you can easily see what failed and look at the log and history for why. If it succeeds, lazylibrarian will move the files into different folder structures for ebook/audiobook/mag as it knows what it asked for, and what it's expecting to find.

**Comment by DarkSir23** [comment]
This makes sense.  I'm going to try that.  

Unrelated, but not really issue-worthy until I look closer:  Since I've added audiobook RSS feeds, I'm catching them in ebook searches.  I've since added m4a and m4b to the filter, but I think I've found some that are mp3 format.  So I wasn't sure if the filter was applying to RSS feeds or not.  I'll also add the word Narrated to the filter, which should catch them from this particular RSS, anyway. :)
